revolutionpi/analog_pin.go

```diff
@@ -1,7 +1,7 @@
 //go:build linux
 
-// Package revolution_pi implements the Revolution Pi board GPIO pins.
-package revolution_pi
+// Package revolutionpi implements the Revolution Pi board GPIO pins.
```

Comment:
had to move these files & change the package name to remove the underscore to appease the linter